### PR TITLE
Grammatical Corrections in messages and comments

### PR DESCRIPTION
**Pull request by shivam-tadas**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
-name = "jsontofile"
-version = "0.1.0"
+name = "jsonsplitter"
```
Comment (biplab5464) [inline Cargo.toml:2]: i didn't like this name 
can you think of a better name

File: readme.md [hunk 2]
```diff
@@ -10,5 +10,4 @@ This Rust version offers significant performance and stability improvements over
 
 [Downlaod Link for windows](https://github.com/biplab5464/jsontofile/blob/main/target/x86_64-pc-windows-gnu/release/jsontofile.exe)
 
-Developed by: Biplab Garita
-
+Originally developed by: Biplab Gartia
```
Comment (biplab5464) [inline readme.md:13]: add your name too under co-developer

File: Cargo.toml (hunk 1 above)
Comment (shivam-tadas) [reply Cargo.toml:2]: Sorry, I can't think of any better name.

File: readme.md (hunk 2 above)
Comment (shivam-tadas) [reply readme.md:13]: Ok, will edit the readme file

File: Cargo.toml (hunk 1 above)
Comment (biplab5464) [reply Cargo.toml:2]: ok i understand

File: readme.md (hunk 2 above)
Comment (biplab5464) [reply readme.md:13]: nice

**Pull request merged**